proposals/dao/vip_9.ts

```diff
@@ -0,0 +1,96 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner } from '@test/helpers';
+
+/*
+
+Timelock Proposal #9
+
+Description: Add DAI, FEI and USDC Compound PCV Deposits to the PCV Guardian
+
+Steps:
+  1 - Deploy all Compound PCV Deposits
+  2 - Add them to the PCV Guardian whitelist
+
+*/
+
+const vipNumber = '9';
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const daiCompoundPCVDeposit = await (
+    await ethers.getContractFactory('ERC20CompoundPCVDeposit')
```

Comment:
Very minor but I'd probs just get the factory once, store in a local var and reuse:

`const ERC20CompoundDepositFactory = await ethers.getContractFactory('ERC20CompoundPCVDeposit');`

Fine as is, just makes it more DRY